README.md

```diff
@@ -33,6 +33,8 @@ roles responsible for building and running apps and operating infrastructure.
 
 For more details and user stories, see [introduction.md](./introduction.md).
 
+[Relations between various OAM resources](./assets/relations_between_various_OAM_resources.png) 
```

Comment:
Hi, please update your contribution branch, it seems quite out-of-date.

Also, I'd suggest to create a `20200727-spec-v0.2.0-data-model.md` under https://github.com/oam-dev/spec/tree/master/design, and then link this file at the end of section: https://github.com/oam-dev/spec/blob/master/2.overview_and_terminology.md#overview-of-the-model

Reply:
@resouer According to you I have to create a .md file named `20200727-spec-v0.2.0-data-model` and add wireframe in this .md file. And next link this `20200727-spec-v0.2.0-data-model.md` file at the end of  https://github.com/oam-dev/spec/blob/master/2.overview_and_terminology.md#overview-of-the-model this file. Right?

Reply:
Yes, the idea is we need to have a `.md` doc to "carry" this diagram, and the best place to reference this `.md` doc is the `overview-of-the-model` section.